android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothSwpSBF77.java

```diff
@@ -49,35 +50,6 @@ public String driverName() {
         return deviceName;
     }
 
-    @Override
-    public void onBluetoothNotify(UUID characteristic, byte[] value) {
-        if (characteristic.equals(CHARACTERISTIC_SBF77_USER_LIST)) {
-            handleVendorSpecificUserList(value);
-        }
-        else {
-            super.onBluetoothNotify(characteristic, value);
-        }
-    }
-
-    @Override
-    protected void setNotifyVendorSpecificUserList() {
-        if (setNotificationOn(SERVICE_CUSTOM_SBF77, CHARACTERISTIC_SBF77_USER_LIST)) {
-            Timber.d("setNotifyVendorSpecificUserList() OK");
-        }
-        else {
-            Timber.d("setNotifyVendorSpecificUserList() FAILED");
-        }
-    }
-
-    @Override
-    protected synchronized void requestVendorSpecificUserList() {
-        BluetoothBytesParser parser = new BluetoothBytesParser();
-        parser.setIntValue(0x00, FORMAT_UINT8);
-        writeBytes(SERVICE_CUSTOM_SBF77, CHARACTERISTIC_SBF77_USER_LIST,
-                parser.getValue());
-        stopMachineState();
-    }
-
```

Comment:
You are accidentally reverting changes that I did in my main branch.
Please exclude this block from your commit;

Reply:
Yes. If I do a commit it will overwrite line history. Therefore, I think is best that you do a commit solving this to keep track of the author.

Reply:
https://github.com/krisjans/openScale/pull/4#discussion_r711718923